tools/topology/topology2/platform/intel/bt-generic.conf

```diff
@@ -245,8 +244,7 @@ Object.PCM.pcm [
 			direction   "capture"
 			name	$BT_CP_PCM_CAPS
 			formats 'S16_LE'
-			rate_min 8000
-			rate_max 48000
+			rates '8000,16000,48000'
```

Comment:
The range previously included 16, 32, 44.1 at least, is it valid to only pick these three of of the range?

Reply:
yes, lets keep it simple.

Reply:
@ujfalusi These match the supported Bluetooth profiles. 8 and 16 for HFP, and 48 for A2DP.

Reply:
@kv2019i isn't 32kHz supported in the newer profiles with BT LE?

Reply:
A bit complicated case for tplg parser.
8000/16000 + mono + S16_LE is one of supported cases
48000 + stereo + S16_LE is another supported case?

sof_tplgreader.py will set 48000 first to rate when 48000 is available in rates, so this will fail again.
> aplay   -Dhw:0,2 -r 48000 -c 1 -f S16_LE -d 5 /dev/zero -v -q

before https://github.com/thesofproject/sof-test/pull/1174 merged, rate is from rate_min, it was working with below,
> aplay   -Dhw:0,2 -r 8000 -c 1 -f S16_LE -d 5 /dev/zero -v -q

Reply:
@fredoh9 explained this to me and I think the key issue is: assuming that rate and number of channels are independent of each other.

Reply:
@plbossart yes, LE will have more rates, but we those also require addition of new SSP configurations which are  not there yet, so until those added, we should advertise what we actually support (and that is 8/16/48 now)

@fredoh9 ack. this is just hitting the limits of what we can express in capabilities. the topology description of PCM cannot express the PCM does not support all combinations of supported channels and rates.